### DISCUSSION TITLE
Why doesn't `run_sync` work on latest Chromes?

### DISCUSSION THREAD

**Discussion by CNSeniorious000** [pyodide/pyodide#5085] [Q&A] [answered]
Recently I found that `run_sync` no longer works on latest versions of Chrome or Chromium-based browsers, while it still works in older ones. Is it related to #4883?

(I am using `0.26.2`)

**Comment by ryanking13** [comment, marked as answer]
> Is it related to https://github.com/pyodide/pyodide/issues/4883

Exactly. Chrome changed JSPI API.

**Reply by CNSeniorious000** [reply]
So sad 🥺
